mongodb/intro/date-aggregation/aggregation-with-date-expression-operators-ii.md

```diff
@@ -0,0 +1,172 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+  - basic
+  
+aspects:
+  - introduction
+  - new
+
+type: normal
+
+category: how to
+
+---
+
+# Aggregation With `$year`, `$month` and `$dayOfMonth`
+
+---
+## Content
+
+Now that we know how to extract the date into an ISODate format, we can use some aggregation operators to find the exact time that the document was made as precise as in milliseconds.
+
+**Note:** For easier readability and comprehension of date operators, we will be using the date from the `name:"Bulbasaur"` document shown in the previous insight. **(The document will also be in the footnotes of every insight of this workout under Previous Document[1])**
+
+**Note:** All 10 date operators have the same syntax:
+
+```js
+{
+  $DATE_OPERATOR: <expression>
+}
+```
+
+### $year
+
+The `$year` operator is used to find the year the document was created.
+
+Since we are only looking for the date of a specific document, we will use the `$match` stage to only match the document we want. Also, we will use the `$project` stage to only aggregate the parts of the date we want.
+
+Example:
+```javascript
+db.pokedex.aggregate([
+  { $match: { name: "Bulbasaur" } },
+  {
+    $project: {
+      year: { $year: "$Date" }
+    }
+  }
+]);
+
+```
+Output:
+```javascript
+{
+  "_id": ObjectId("5d9d8a5a0b24990f19398208"),
+  "year" : 2019
+}
+```
+
+As you can see in the example above, the `$match` stage was used to match the pokémon(document) named "Bulbasaur" and get the year it was created.
```

Comment:
```suggestion
As you can see in the example above, the `$match` stage was used to match the pokémon(document) named `"Bulbasaur"` and get the year it was created.
```

Reply:
Also why collectionName(document) format?

> pokémon(document)

Reply:
I don't think  it references the format, but rather that a pokemon is a document. I can change it to `pokemon (a document)` or just remove the parentheses entirelly. What do you think?

Reply:
```suggestion
As you can see in the example above, the `$match` stage was used to match the pokémon document named `"Bulbasaur"` and get the year it was created.
```